sympy/parsing/autolev/test-examples/pydy-example-repo/non_min_pendulum.py

```diff
@@ -33,3 +34,7 @@
 config[0] = f_c
 zero = zero.row_insert(zero.shape[0], sm.Matrix([[0]]))
 zero[zero.shape[0]-1] = config[0]
+# Works
+# Numerical parts for such kind of things won't work.
+# The current numerical parts only work with the simple kane's method use (no_args),
+# no config constraints etc. Have to add auxiliary stuff later.
```

Comment:
An issue should be opened about supporting config constraints, auxiliary speeds, etc.

Reply:
Auxiliary speeds have to be added and I will open an issue for that.
The problem with config constraints is that they aren't formally specified in Autolev unlike motion constraints. So, they cannot be determined for the Kane's method object.
Can we ask the users to use a made up command like ConfigConstraint() or something?

Reply:
Configuration constraint issue: #15172.